build_tools/python/e2e_test_framework/definitions/iree_definitions.py

```diff
@@ -144,7 +144,7 @@ class MLIRDialectType(Enum):
   """Imported MLIR dialect type."""
   NONE = "none"
   TOSA = "tosa"
-  MHLO = "mhlo"
+  STABLE_HLO = "stablehlo_experimental"
```

Comment:
Nit: Consider spelling this as "STABLEHLO" (without an underscore), since the canonical name in both prose and code doesn't include whitespaces or underscores (i.e. "StableHLO" / "stablehlo").

Reply:
Updated to be a single word.